### PR TITLE
Updates Pulp Plugin Day description

### PR DESCRIPTION
**Pull request by dkliban**
Please don't merge yet. I want some feedback from other Pulp devs.

### CODE REVIEW

File: content/fringes/pulppluginday.md [hunk 1]
```diff
@@ -2,20 +2,36 @@
 title: "Pulp Plugin Day"
 date: 2018-11-23
 eventdate: "6th February 2019"
-draft: true
+draft: false
 ---
 Pulp is a platform for managing repositories of software packages, but it can store any kind of
```
Comment (bmbouter) [inline content/fringes/pulppluginday.md]: Want to unline-wrap this post?

Comment (bmbouter) [reply content/fringes/pulppluginday.md]: This sentence I think would work better with an 'and' in the middle (compound predicate). Maybe

Pulp is a platform for managing repositories of software packages and can store any kind of binary data (e.g. cat photos).

File: content/fringes/pulppluginday.md [hunk 2]
```diff
@@ -2,20 +2,36 @@
...
 binary data (e.g. cat photos). The core provides a REST API and manages files, but the real value
 for users is delivered by the plugins which allow core to manage a content type like a cat photo.
 
 Please join some of the Pulp developers on February 6th for Pulp Plugin Day. This will be an
```
Comment (bmbouter) [inline content/fringes/pulppluginday.md]: s/Please/You are invited to/

File: content/fringes/pulppluginday.md [hunk 3]
```diff
@@ -2,20 +2,36 @@
...
-opportunity to add new features to any of the
+opportunity to learn how to use Pulp 3, add new features to any of the
```
Comment (bmbouter) [inline content/fringes/pulppluginday.md]: s/any of the/an/

File: content/fringes/pulppluginday.md [hunk 4]
```diff
@@ -2,20 +2,36 @@
...
 [existing plugins](https://pulpproject.org/pulp-3-plugins/) or start on a new plugin to support
```
Comment (bmbouter) [inline content/fringes/pulppluginday.md]: need a comma before the `or` to make it the Oxford comma

File: content/fringes/pulppluginday.md [hunk 5]
```diff
@@ -2,20 +2,36 @@
...
-Then everyone is going to setup a development environment and start working on a Pulp plugin of
-their choice. 
-
 Please remember to [register](https://registration.cfgmgmtcamp.be/ghent/2019/) through the general Config Management Camp registration system.
+
+We are going to start at 10:00 with a brief introduction to Pulp. We will then help everyone setup
+a development environment on their laptop. At that point participants will decide what they would
+like to focus on. Some possible activities include:
+
+ * Fix an existing issue in [pulp.plan.io](https://pulp.plan.io/issues/)
```
Comment (bmbouter) [inline content/fringes/pulppluginday.md]: I think we should go into Redmine and ensure that there are good issue with the tags 'pulp3' + 'Easy Fix' and link to that here. Linking here now is fine, and someone can go fix the data closer to the event maybe? Also what about a link for 'groomed=True, state=NEW, tracker=[Story,Task]'. Anyway more specific Redmine links I think would be good.

File: content/fringes/pulppluginday.md [hunk 6]
```diff
@@ -2,20 +2,36 @@
...
+ * Follow the [Plugin Writer's Guide](https://docs.pulpproject.org/en/3.0/nightly/plugins/index.html#plugin-writer-s-guide) to plan support for a new content type
+ * Learn how to use the [Plugin API](https://docs.pulpproject.org/en/pulpcore-plugin/nightly/) to implement support for a new content type
+ * File bug reports or feature requests on [pulp.plan.io](https://pulp.plan.io/issues/)
+ * Learn how to use Pulp's [REST API](https://docs.pulpproject.org/en/3.0/nightly/integration-guide/rest-api/#pulpcore-rest-api) to interact with existing plugins
+
+Participants can break out into groups or work on their own. Pulp developers will be available
+to answer questions and help troubleshoot problems, but everyone is encouraged to help each other.
+
+Participants that want to learn more than the REST API should be comfortable with writing Python code.
+Pulp development environment is most easily deployed using Vagrant. Participants are encouraged to
```
Comment (bmbouter) [inline content/fringes/pulppluginday.md]: Is there any way we can link to the setup instruction in our docs right from here?
Also if we are directly recommending Vagrant here, can we link to their install instructions?
Also what about mentioning the Ansible installer that should be an option offered too?
I think having a lot of 'choice' about how to install will really help this effort.

File: content/fringes/pulppluginday.md [hunk 7]
```diff
@@ -2,20 +2,36 @@
...
+pre-install Vagrant on their machines prior to the event. The development environment can be deployed
+using either the libvirt or docker backend for Vagrant.
```
Comment (bmbouter) [inline content/fringes/pulppluginday.md]: Can we lead w/ docker (first) here?

File: content/fringes/pulppluginday.md [hunk 8]
```diff
@@ -2,20 +2,36 @@
 title: "Pulp Plugin Day"
 date: 2018-11-23
 eventdate: "6th February 2019"
-draft: true
+draft: false
 ---
 Pulp is a platform for managing repositories of software packages, but it can store any kind of
 binary data (e.g. cat photos). The core provides a REST API and manages files, but the real value
 for users is delivered by the plugins which allow core to manage a content type like a cat photo.
```
Comment (ipanova) [inline content/fringes/pulppluginday.md]: s/allow core/enable core/

**Comment by toshywoshy** [comment]
Where are we on this PR?

**Pull request closed without merging**